doc/manual/manual.tex

```diff
@@ -6656,7 +6656,71 @@ \subsubsection{Crustal deformation}
+
+\lstinputlisting[language=prmfile]{cookbooks/continental_extension/continental_extension_material_model.prm}
+
+The cookbook below provides one such example where the continental lithosphere undergoes extension.  Notably, the model design follows that of numerous previously published continental extension studies  \cite[and references therein]{Hui11,Bru14,Nal15}.
+
+\paragraph{Continental Extension}
+The 2D cartesian model spans 400 (x) by 100 (y) km and has a finite element grid with uniform 2 km spacing. Unlike the crustal deformation cookbook, the mesh is not refined with time.
+
+\lstinputlisting[language=prmfile]{cookbooks/continental_extension/continental_extension_geometry_mesh.prm}
+
+
+Similar to the crustal deformation examples above, this model contains a free surface.  Deformation is driven by constant horizontal (x-component) velocities (0.25 cm/yr) on the side boundaries (y-velocity component unconstrained), while the bottom boundary has vertical inflow to balance the lateral outflow.  The top and bottom boundaries have fixed temperatures, while the sides are insulating (zero heat flux).
+
+\lstinputlisting[language=prmfile]{cookbooks/continental_extension/continental_extension_boundary_conditions.prm}
+
+Sections of the lithosphere with distinct properties are represented by compositional fields for the upper crust (20 km thick), lower crust (10 km thick) and mantle lithosphere (70 km thick).  A mechanically weak seed within the mantle lithosphere helps localize deformation.  Material (viscous flow law parameters, cohesion, internal friction angle) and thermodynamic properties for each compositional field are based largely on previous numerical studies.   Dislocation creep viscous flow parameters are taken from published deformation experiments for wet quartzite \cite{RB04}, wet anorthite \cite{RGWD06} and dry olivine \cite{HK04}. 
+
+\lstinputlisting[language=prmfile]{cookbooks/continental_extension/continental_extension_composition.prm}
+
+The initial thermal structure, radiogenic heating model and associated thermal properties are consistent with the prescribed thermal boundary conditions and produce a geotherm characteristic of the continental lithosphere.  The equations defining the initial geotherm \cite{Cha86} follow the form
+
+\begin{align}
```

Comment:
the formula here is part of the sentence structure. As such, you should avoid the empty lines before and after the formula because that tells latex to start a new paragraph, with all of the indentation that goes along with it.